README.md

```diff
@@ -49,12 +49,15 @@ You request access to the anonymised toy dataset from
 
 ## Required packages
 * R (>= 3.1.0),
-* XML,
 * data.table,
-* yaml,
+* dbplyr,
```

Comment:
This isn't mentioned earlier? Typo?

Reply:
dbplyr must be installed on the system. But you never explicitly use it. It is a bit strange that way